openshift/04-deployment-template.yml

```diff
@@ -77,10 +77,12 @@ objects:
           - name: THREESCALE_DEPLOYMENT_ENV
             value: "${ENVIRONMENT}"
           - name: THREESCALE_PORTAL_ENDPOINT
-            value: "http://apicast-mapping-service/config"
-          image: "${APICAST_IMAGE}"
+            value: "http://apicast-mapping-service-${RELEASE_REF}/config"
+          - name: APICAST_OIDC_LOG_LEVEL
+            value: "${APICAST_OIDC_LOG_LEVEL}"
```

Comment:
I think this does not need to be a Parameter.

Reply:
Ok So I will place the env var with notice as default.